namespaces/live-1.cloud-platform.service.justice.gov.uk/pathfinder-dev/02-limitrange.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: LimitRange
+metadata:
+  name: limitrange
+  namespace: pathfinder-dev
+spec:
+  limits:
+  - default:
+      cpu: 360m
+      memory: 1024Mi
+    defaultRequest:
```

Comment:
The CPU limit request here seems excessive. This is going to reserve almost half a CPU for every container you launch. Please reduce this to 10m

See: https://user-guide.cloud-platform.service.justice.gov.uk/concepts.html#namespace-container-resource-limits